scripts/start.sh

```diff
@@ -473,13 +470,13 @@ elif [ "$MODE" == "local_basic" ]; then
             sleep $DB_SLEEP
         fi
     fi
-    /concordium-node --baker-credentials-file "${DATA_DIR}/baker-${BAKER_ID}-credentials.json" --no-dnssec $ARGS --id $(printf "%016d\n" $BAKER_ID)
+    /concordium-node --baker-credentials-file "/genesis-data/genesis-${NUM_BAKERS}-bakers/bakers/baker-${BAKER_ID}-credentials.json" --no-dnssec $ARGS --id $(printf "%016d\n" $BAKER_ID)
 elif [ "$MODE" == "local_bootstrapper" ]; then
     export NODE_ID="0000000001000000"
     /p2p_bootstrapper-cli \
         --id $NODE_ID \
         --listen-port 8888 \
-        --regenesis-block-hashes-file $DATA_DIR/genesis_hash $EXTRA_ARGS
+        --regenesis-block-hashes-file $DATA_DIR/genesis_hash $ARGS $EXTRA_ARGS
```

Comment:
Lines 213-216:
```
if [ -n "$EXTRA_ARGS" ];
then
    ARGS="$ARGS $EXTRA_ARGS"
fi
```
so this should be just `$ARGS`

Reply:
This is merged, commenting here is not useful. Just fix it.